provider/pkg/gen/schema.go

```diff
@@ -231,6 +241,7 @@ func PulumiSchema(swagger map[string]interface{}) pschema.PackageSpec {
 
 	csharpNamespaces := map[string]string{
 		"helm.sh/v3": "Helm.V3",
+		"":           "Provider",
```

Comment:
A little confused why this was needed? I see that Provider.cs already existed right?

Reply:
I couldn't get the C# SDK to generate correctly without that. The compatibility logic is causing problems without it.

Reply:
Do we have a tracking bug for this in codegen?